2020-11-22 22:54:11.179700: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcuda.so.1
2020-11-22 22:54:13.362086: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1561] Found device 0 with properties: 
pciBusID: 0000:1a:00.0 name: Tesla V100-SXM2-16GB computeCapability: 7.0
coreClock: 1.53GHz coreCount: 80 deviceMemorySize: 15.75GiB deviceMemoryBandwidth: 836.37GiB/s
2020-11-22 22:54:13.367139: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
2020-11-22 22:54:13.418110: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcublas.so.10
2020-11-22 22:54:13.450936: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcufft.so.10
2020-11-22 22:54:13.470847: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcurand.so.10
2020-11-22 22:54:13.556683: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcusolver.so.10
2020-11-22 22:54:13.576882: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcusparse.so.10
2020-11-22 22:54:13.755949: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudnn.so.7
2020-11-22 22:54:13.758897: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1703] Adding visible gpu devices: 0
2020-11-22 22:54:13.759350: I tensorflow/core/platform/cpu_feature_guard.cc:143] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 AVX512F FMA
2020-11-22 22:54:13.776980: I tensorflow/core/platform/profile_utils/cpu_utils.cc:102] CPU Frequency: 2400000000 Hz
2020-11-22 22:54:13.777400: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x564ee1229c40 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-11-22 22:54:13.777444: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-11-22 22:54:13.779039: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1561] Found device 0 with properties: 
pciBusID: 0000:1a:00.0 name: Tesla V100-SXM2-16GB computeCapability: 7.0
coreClock: 1.53GHz coreCount: 80 deviceMemorySize: 15.75GiB deviceMemoryBandwidth: 836.37GiB/s
2020-11-22 22:54:13.779116: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
2020-11-22 22:54:13.779134: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcublas.so.10
2020-11-22 22:54:13.779150: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcufft.so.10
2020-11-22 22:54:13.779165: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcurand.so.10
2020-11-22 22:54:13.779181: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcusolver.so.10
2020-11-22 22:54:13.779195: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcusparse.so.10
2020-11-22 22:54:13.779211: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudnn.so.7
2020-11-22 22:54:13.783088: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1703] Adding visible gpu devices: 0
2020-11-22 22:54:13.786360: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
2020-11-22 22:54:13.933285: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1102] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-11-22 22:54:13.933356: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1108]      0 
2020-11-22 22:54:13.933814: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1121] 0:   N 
2020-11-22 22:54:13.937619: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1247] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 14865 MB memory) -> physical GPU (device: 0, name: Tesla V100-SXM2-16GB, pci bus id: 0000:1a:00.0, compute capability: 7.0)
2020-11-22 22:54:13.949289: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x564ee1c888b0 initialized for platform CUDA (this does not guarantee that XLA will be used). Devices:
2020-11-22 22:54:13.949332: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Tesla V100-SXM2-16GB, Compute Capability 7.0
WARNING:tensorflow:Model was constructed with shape (None, 21, 11, 15, 1) for input Tensor("input_1:0", shape=(None, 21, 11, 15, 1), dtype=float32), but it was called on an input with incompatible shape (None, 10, 11, 15, 1).
WARNING:tensorflow:Model was constructed with shape (None, 21, 11, 15, 1) for input Tensor("input_1:0", shape=(None, 21, 11, 15, 1), dtype=float32), but it was called on an input with incompatible shape (None, 10, 11, 15, 1).
2020-11-22 22:54:21.739879: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcublas.so.10
2020-11-22 22:54:22.504103: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudnn.so.7
WARNING:tensorflow:Model was constructed with shape (None, 21, 11, 15, 1) for input Tensor("input_1:0", shape=(None, 21, 11, 15, 1), dtype=float32), but it was called on an input with incompatible shape (None, 10, 11, 15, 1).
2020-11-22 23:00:35.836312: W tensorflow/python/util/util.cc:329] Sets are not currently considered sequences, but this may change in the future, so consider avoiding using them.
WARNING:tensorflow:From /nas/longleaf/home/lswhiteh/.conda/envs/blinx/lib/python3.6/site-packages/tensorflow/python/ops/resource_variable_ops.py:1817: calling BaseResourceVariable.__init__ (from tensorflow.python.ops.resource_variable_ops) with constraint is deprecated and will be removed in a future version.
Instructions for updating:
If using Keras pass *_constraint arguments to layers.
WARNING:tensorflow:Model was constructed with shape (None, 21, 11, 15, 1) for input Tensor("input_1:0", shape=(None, 21, 11, 15, 1), dtype=float32), but it was called on an input with incompatible shape (None, 10, 11, 15, 1).
